core/templates/dev/head/css/oppia.css

```diff
@@ -4166,3 +4166,7 @@ md-card.preview-conversation-skin-supplemental-card {
   right: -25px;
   top: 175px;
 }
+
+.conversation-skin-tutor-card-content {
```

Comment:
Hi @AllanYangZhou -- looks great! One thing though. The conversation-skin classes are located in core/templates/dev/head/player/conversation_skin_directive.html. Can this rule be moved there instead?

Thanks!